operations/hardware_state_manager/Lock_and_Unlock_Management_Nodes.md

```diff
@@ -88,11 +88,14 @@ The return value of the script is 0 if locking was successful. Otherwise, a non-
 
 Use the `cray hsm locks lock` command to perform locking.
 
-* To lock all nodes with the _Management_ role.
+***NOTE: When locking NCNs, you must lock their NodeBMCs as well.***
+
+##### To lock all nodes (and their BMCs) with the _Management_ role.
 
    The *processing-model rigid* parameter means that the operation must succeed on all
    target nodes or the entire operation will fail.
 
+1. Lock the management nodes
```

Comment:
Add a period at the end of all of the steps below.

Reply:
Done.